src/components/plots/xyPlot.ts

```diff
@@ -7,6 +7,9 @@ module Plottable {
     private _autoAdjustXScaleDomain = false;
     private _autoAdjustYScaleDomain = false;
 
+    private _adjustYDomainOnChangeFromXFunctionWrapper: Function;
+    private _adjustXDomainOnChangeFromYFunctionWrapper: Function
```

Comment:
Type the function to be more specific

Also, missing semicolon (tslint should've warned against this)